RecFCCeeCalorimeter/src/components/CaloTowerToolFCCee.cpp

```diff
@@ -357,15 +349,16 @@ void CaloTowerToolFCCee::CellsIntoTowers(std::vector<std::vector<float>>& aTower
       float cellPhi   = atan2(cellY, cellX);
       iTheta = idTheta(cellTheta);
       iPhi = idPhi(cellPhi);
-      if (aType == SegmentationType::kPhiTheta) {
-        aTowers[iTheta][phiNeighbour(iPhi)] += cell.getEnergy() * sin(segmentationHCal->theta(cell.getCellID()));
-      } else {
-        aTowers[iTheta][phiNeighbour(iPhi)] += cell.getEnergy() * sin(segmentation->theta(cell.getCellID()));
-      }
+      aTowers[iTheta][phiNeighbour(iPhi)] +=
+	//        cell.getEnergy() * sin(segmentation->theta(cell.getCellID()));
```

Comment:
remove the commented line. 
The change assumes that cells are already positioned and we get theta directly from the cell positions rather than their cellID, but this hypothesis was anyway already used in other parts of the code before (to calculate iTheta and iPhi from cellTheta and cellPhi) so I think we can go ahead with this change and simplify the code by removing the need of passing the segmentation class to the cellsInTowers method

Reply:
Commented line removed